ami/ml/tasks.py

```
@@ -95,25 +95,49 @@ def remove_duplicate_classifications(project_id: int | None = None, dry_run: boo
     return num_deleted
 
 
-@celery_app.task(soft_time_limit=10, time_limit=20)
+# Timeout per sync service in the periodic beat task. Shorter than the default (90s for
+# cold-start waits) since a missed check just waits for the next beat cycle.
+# Worst case: 4 attempts (initial + 3 retries) × 8s timeout + backoff (0 + 2 + 4) = 38s per service.
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)